.vscodeignore

```diff
@@ -8,3 +8,5 @@ src/**
 test/**
 rust/**
 guide/**
+
+../**/*
```

Comment:
Does this have an effect? My understanding was that `.vscodeignore` only applied to files within the directory so am curious how this works.